test/functionalTest/cases/2995_q_in_subscriptions/q_value_non-compatible_sub_LTE.test

```diff
@@ -0,0 +1,176 @@
+# Copyright 2017 Telefonica Investigacion y Desarrollo, S.A.U
```

Comment:
Just to know... have you tested new tests in both cache and no-cache testHarness.sh variants?

Reply:
Ah, no.
I never remember that 'detail' ... :-(
What I have done is func test AND valgrind test in both Ubuntu 17.04 and CentOS 6.5

Reply:
Most of the cases is not so needed and the check jenkins does is ok for the few cases in may cause a problem. However, this PR is special as it is related with filter evaluation on csubs cache, and we have found in the past some cases in which something working for csubs cache was not working for no-cache an the other way around.

In sum: in this case is worth the check, in other cases is not needed.

Reply:
Entire test suite tested OK with noCache

Reply:
OK, NTC.